horreum-integration/pom.xml

```diff
@@ -0,0 +1,185 @@
+                                    <entry key="horreum.build.commit" value="${buildNumber}" />
+                                    <entry key="horreum.build.timestamp" value="${timestamp}" />
+                                </propertyfile>
+                            </target>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.2.0</version>
+            </plugin>
+        </plugins>
+    </pluginManagement>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <executions>
+                <execution>
+                    <goals>
+                        <goal>test-jar</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+                <systemPropertyVariables>
+                    <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+                    <maven.home>${maven.home}</maven.home>
+                    <horreum.commit.id>${buildNumber}</horreum.commit.id>
+                </systemPropertyVariables>
+                <additionalClasspathElements>
+                     <additionalClasspathElement>../horreum-client/target/horreum-client-${project.version}-tests.jar</additionalClasspathElement>
+                </additionalClasspathElements>
+            </configuration>
+        </plugin>
+        <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>buildnumber-maven-plugin</artifactId>
+        </plugin>
+    </plugins>
+
+    <testResources>
+        <testResource>
+            <directory>src/test/resources</directory>
+            <filtering>false</filtering>
+            <includes>
+                <include>**/*</include>
+            </includes>
+            <excludes>
+                <exclude>*/*-horreum.yaml</exclude>
+            </excludes>
+        </testResource>
+        <testResource>
+            <directory>../infra</directory>
+            <targetPath>docker-compose</targetPath>
+            <includes>
+                <include>*.env</include>
+                <include>*.sh</include>
+                <include>*.yml</include>
+            </includes>
+        </testResource>
+        <testResource>
+            <directory>../horreum-backend/src/main/resources</directory>
+            <targetPath>docker-compose</targetPath>
+            <includes>
+                <include>keycloak-horreum.json</include>
+            </includes>
+        </testResource>
+        <testResource>
```

Comment:
Just add another include above?
I also wonder, since referencing path another module confuses IntelliJ if it wouldn't be better to just replace it with a symlink...

Reply:
keycloak-horreum.json is in backend resources only to become part of the image that's reused as Keycloak init-container in Openshift; we can do better.